std/math.d

```diff
@@ -7344,6 +7551,14 @@ if (isUnsigned!F && isUnsigned!G && isUnsigned!H)
     return result;
 }
 
+///
+@safe pure nothrow @nogc unittest
+{
+    assert(powmod(1U, 10U, 3U) == 1);
+    assert(powmod(3U, 2U, 6U) == 3);
+    assert(powmod(5U, 5U, 15U) == 5);
```

Comment:
Two or three more examples would be nice here, especially ones for which it gets clear that's `x^y % m`, e.g.

```d
    assert(powmod(2U, 3U, 5U) == 3);
    assert(powmod(2U, 4U, 5U) == 1);
    assert(powmod(2U, 5U, 5U) == 2);
```